docs(testing): grammar and clarity improvements (#2685)
Permify/permify: docs/getting-started/testing.mdx

title: 'Testing & Validation'
---

> Testing is a critical process when building and maintaining an authorization system. This page explains how to ensure the new authorization model and related authorization data works as expected in Permify.

Reviewer: ```suggestion
Testing is a critical process when building and maintaining an authorization system. This page explains how to ensure your authorization model and data works as expected in Permify.
```
[suggestion]: reduce duplication of 'authorization' in this paragraph. 3 times is probably too many in such a small paragraph!
